package.json

```diff
@@ -30,23 +30,23 @@
   "homepage": "https://trails-game.com",
   "devDependencies": {
     "@babel/core": "^7.18.5",
+    "@babel/eslint-parser": "^7.18.2",
     "@babel/plugin-transform-async-to-generator": "^7.17.12",
     "@babel/plugin-transform-runtime": "^7.18.5",
     "@babel/preset-env": "^7.18.2",
-    "@babel/eslint-parser": "^7.18.2",
-    "babel-loader": "^8.2.5",
+    "babel-loader": "^9.1.2",
     "eslint": "^8.18.0",
     "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^6.0.0",
     "webpack": "^5.73.0",
-    "webpack-cli": "^4.10.0"
+    "webpack-cli": "^5.1.1"
   },
   "dependencies": {
     "3d-force-graph": "^1.70.10",
```

Comment:
This is the core compoent that needs a bump

Reply:
The lockfile has been changed for that already. A `npm install` or `npm ci` will produce an install with version `1.71.4` of `3d-force-graph`.